AR-Top-MobileApp/Assets/Prefabs/MapController.cs

```diff
@@ -71,7 +69,7 @@ void Start ()
 		string dataAsJson = "";
 
 		// Load example file
-		string filePath = Path.Combine(Application.streamingAssetsPath, "TestData/voxel_shell_8x8.json");
+		string filePath = Path.Combine(Application.streamingAssetsPath, "TestData/voxel_cube_8x8.json");
```

Comment:
This change can probably be removed.

Reply:
This does have some uses for testing purposes but I commented it out.